components/automate-ui/src/app/modules/team/team-add-users/team-add-users.component.ts

```diff
@@ -169,9 +169,7 @@ export class TeamAddUsersComponent implements OnInit, OnDestroy {
   }
 
   getConfirmBtnText(): string {
-    if (this.usersNotFiltered().length === 0) {
-      return '';
-    } else if (this.addingUsers) {
+    if (this.addingUsers) {
```

Comment:
this was a bonus fix. I noticed in the case of `if (this.usersNotFiltered().length === 0)`, the `Add Users` button just became a shortened blank button which looked off to me. So now when there are no users left to add you'll see a disabled `Add Users` button

Reply:
oh was ur chef-ui-library up to date? those buttons should be totally hidden if there are no more users left to add.

Reply:
@susanev is correct; this change should not be needed.

Reply:
oh good catch thanks!